types/yauzl-promise.d.ts

```diff
@@ -0,0 +1,6 @@
+import * as yauzl from 'yauzl-promise'
+declare module 'yauzl-promise' {
+  export interface ZipFile extends yauzl.ZipFile {
+    [Symbol.asyncIterator]()
+  }
```

Comment:
[I just patched this upstream](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/68387), so we can update `@types/yauzl-promise` and delete this.